wallet.go

```diff
@@ -132,3 +132,29 @@ func Walletpassphrase(url string, params []string) (err error) {
 	}
 	return
 }
+
+// Listwallets returns the list of loaded wallets.
+func ListWallets(url string, params []string) (wallets []string, err error) {
+	byteWallets, err := SendRequest(url, types.MethodListWallets, params)
+	if err != nil {
+		return
+	}
+	err = json.Unmarshal(byteWallets, &wallets)
+	if err != nil {
+		return
+	}
+	return
+}
+
+// Listreceivedbyaddress returns a list of transactions
+func ListReceivedByAddress(url string, params []string) (receivedTx []types.ListReceivedByAddressResult, err error) {
```

Comment:
should be plural since an array is returned, e.g. `receivedTXs` or even better `listReceivedByAddressResults`